proxy.py

```
@@ -1108,6 +1120,48 @@ async def scaffolding_apply(body: ScaffoldRequest, auth: AuthContext = Depends(v
 
 
+# ─── Skill Registry Refresh ───────────────────────────────────────────────
+
+class SkillsRefreshRequest(BaseModel):
+    force: bool = Field(default=False, description="Bypass 1-hour TTL and force-refresh")
+
+
+@app.post("/agent/skills/refresh")
+async def skills_refresh(
+    body: SkillsRefreshRequest,
+    auth: AuthContext = Depends(verify_api_key),
```

Ruff (0.15.15)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)